src/BusinessLogic/Resources/js/FooterController.js

```diff
@@ -54,15 +56,15 @@ var Packlink = window.Packlink || {};
             systemInfoPanel.classList.add('loading');
             debugStatus = !debugStatus;
 
-            ajaxService.post(config.setStatusUrl, {status: debugStatus}, function (response) {
+            ajaxService.post(config.setDebugStatusUrl, {status: debugStatus}, function (response) {
```

Comment:
Same above: `.setDebugStatusUrl `

Reply:
Same as above. This one is needed to turn the debug mode on and off.